alerts/system_alerts.libsonnet

```diff
@@ -98,22 +98,22 @@
               severity: 'critical',
             },
             annotations: {
-              message: 'The API server has a 99th percentile latency of {{ $value }} seconds for {{$labels.verb}} {{$labels.resource}}.',
+              message: 'The API server has a 99th percentile latency of {{ $value }} seconds for {{ $labels.verb }} {{ $labels.resource }}.',
             },
           },
           {
             alert: 'KubeAPIErrorsHigh',
             expr: |||
               sum(rate(apiserver_request_count{%(kubeApiserverSelector)s,code=~"^(?:5..)$"}[5m])) without(instance, %(podLabel)s)
                 /
-              sum(rate(apiserver_request_count{%(kubeApiserverSelector)s}[5m])) without(instance, pod) * 100 > 5
+              sum(rate(apiserver_request_count{%(kubeApiserverSelector)s}[5m])) without(instance, pod) * 100 > 10
```

Comment:
sure. this needs to be changed because this alert is an exact duplicate of the `warning` alert that follows. One of them should have a different rate.